Subject: use stage1flex from flex-native to fix cross compiling

When cross compiling, we would have failure because stage1flex cannot run on host.
Use stage1flex from flex-native to fix this problem.

This patch is OE specific as upstream may have other way to fix this cross compilation
problem.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/Makefile.am | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 77ddf4b..09ffddb 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -6,17 +6,10 @@ LIBS = @LIBS@
 m4 = @M4@
 
 bin_PROGRAMS = flex
-noinst_PROGRAMS = stage1flex
 lib_LTLIBRARIES = \
 	libfl.la \
 	libfl_pic.la
 
-stage1flex_SOURCES = \
-	scan.l \
-	$(COMMON_SOURCES)
-
-stage1flex_CFLAGS = $(AM_CFLAGS)
-
 flex_SOURCES = \
 	$(COMMON_SOURCES)
 
@@ -89,8 +82,8 @@ skel.c: flex.skl mkskel.sh flexint.h tables_shared.h tables_shared.c
 	  $(SHELL) $(srcdir)/mkskel.sh > $@.tmp
 	mv $@.tmp $@
 
-stage1scan.c: scan.l stage1flex$(EXEEXT)
-	./stage1flex$(EXEEXT) -o $@ $<
+stage1scan.c: scan.l
+	stage1flex$(EXEEXT) -o $@ $<
 
 # make needs to be told to make parse.h so that parallelized runs will
 # not fail.
-- 
2.1.0

